src/docs/deployment/android.md

```diff
@@ -172,63 +172,11 @@ Configure signing for your app by editing the
 
 Release builds of your app will now be signed automatically.
 
-## Enabling Proguard
+## R8
 
-By default, Flutter does not obfuscate or minify the Android host.
-If you intend to use third-party Java, Kotlin, or Android libraries,
-you might want to reduce the size of the APK or protect that code from
-reverse engineering.
-
-For information on obfuscating Dart code, see
-[Obfuscating Dart Code][] in the [Flutter wiki][].
-
-### Step 1 - Configure Proguard
-
-Create a `/android/app/proguard-rules.pro` file and
-add the rules listed below.
-
-```
-## Flutter wrapper
--keep class io.flutter.app.** { *; }
--keep class io.flutter.plugin.**  { *; }
--keep class io.flutter.util.**  { *; }
--keep class io.flutter.view.**  { *; }
--keep class io.flutter.**  { *; }
--keep class io.flutter.plugins.**  { *; }
--dontwarn io.flutter.embedding.**
-```
-
-This configuration only protects Flutter engine libraries.
-Any additional libraries (for example, Firebase) require adding
-their own rules.
-
-### Step 2 - Enable obfuscation and/or minification
-
-Open the `/android/app/build.gradle` file and locate the `buildTypes`
-definition. Inside the `release` configuration section,
-set the `minifiyEnabled` and `useProguard` flags to true.
-You must also point Proguard to the file you created in step 1:
-
-```
-android {
-
-    ...
-
-    buildTypes {
-
-        release {
-
-            signingConfig signingConfigs.release
-
-            minifyEnabled true
-            useProguard true
-
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
-
-        }
-    }
-}
-```
+[R8]() is the new code shrinker from Google, and it's enabled by default
```

Comment:
This link won't work with empty round brackets. You must either use empty square brackets, or no empty brackets at all. So either `[R8][]` or `[R8]`.

Reply:
Thanks!